amundsen_application/static/js/components/HomePage/tests/index.spec.tsx

```diff
@@ -8,31 +8,23 @@ import SearchBar from 'components/SearchPage/SearchBar';
 import MyBookmarks from 'components/common/Bookmark/MyBookmarks';
 import PopularTables from 'components/common/PopularTables';
 
+import { getMockRouterProps } from 'fixtures/mockRouter';
+
 describe('HomePage', () => {
   const setup = (propOverrides?: Partial<HomePageProps>) => {
+    const mockLocation = {
+      search: '/search?searchTerm=testName&selectedTab=table&pageIndex=1',
+      pathname: 'mockstr',
+      state: jest.fn(),
+      hash: 'mockstr',
+    };
+    const routerProps = getMockRouterProps<any>(null, mockLocation);
```

Comment:
It looks like we only ever need `location.search`. What if instead of overwriting the entire `location` object we only did `location.search`?

Reply:
If we really need to overwrite the whole `location` object, we can do 

```
props = {
  ...routerProps,
  location: {
    // custom overrides
  } 
}
```

Reply:
I can also make it such that `getMockRouterProps` takes `Partial<Location>`, and anything passed in will be used, else use the default.